frontend/pages/policies/PolicyPage/components/PolicyForm/PolicyForm.tsx

```diff
@@ -445,12 +353,83 @@ const PolicyForm = ({
           </div>
         </>
       );
-      /* eslint-enable */
     }
 
     return null;
   };
 
+  const renderPlatformCompatibility = () => {
+    const displayPlatforms = displayOrder
+      .filter((platform) => platform.selected)
+      .map((platform) => {
+        return platform.displayName;
+      });
+
+    return (
+      <span className={`${baseClass}__platform-compatibility`}>
+        {isEditMode ? (
+          <>
+            <b>Checks on:</b>
+            <span className="platforms-text">
+              {displayPlatforms.join(", ")}
+            </span>
+            <span className={`tooltip`}>
+              <span
+                className={`tooltip__tooltip-icon`}
+                data-tip
+                data-for="query-compatibility-tooltip"
+                data-tip-disable={false}
+              >
+                <img alt="question icon" src={QuestionIcon} />
+              </span>
+              <ReactTooltip
+                place="bottom"
+                type="dark"
+                effect="solid"
+                backgroundColor="#3e4771"
+                id="query-compatibility-tooltip"
+                data-html
+              >
+                <span className={`tooltip__tooltip-text`}>
+                  To choose new platforms,
+                  <br />
+                  please create a new policy.
+                </span>
+              </ReactTooltip>
+            </span>
+          </>
+        ) : (
+          <>
+            <b>Compatible with:</b>
```

Comment:
```suggestion
            <b>Checks on:</b>
```

Small nit to use "Checks on:" consistently instead of "Compatible with:"